toolkit/tools/internal/pkgjson/pkgjson.go

```diff
@@ -73,11 +74,54 @@ type Package struct {
 	RunTests      bool          `json:"RunTests"`      // Should we run tests for this package.
 }
 
+//cmp func(a, b E) int
+
+func PackageLess(a, b PackageVer) bool {
```

Comment:
For consistency, I'd consider creating a set of `func (pkgVer *PackageVer) LT|GT|LE|GE|EQ|NEQ` functions like what you did for intervals.

Reply:
Done